core/src/commonMain/kotlin/com/ustadmobile/core/util/ext/PairExt.kt

```diff
@@ -2,27 +2,33 @@ package com.ustadmobile.core.util.ext
 
 import kotlin.math.abs
 
-val validResolutions = listOf(Pair(480,360), Pair(176,144))
+
+/**
+ * Standard Low and High Quality Resolutions as per Android recommended
+ * @ https://developer.android.com/guide/topics/media/media-formats
+ */
+val VALID_RESOLUTIONS = listOf(Pair(480,360), Pair(176,144), Pair(360,480),
+        Pair(144,176), Pair(640, 360), Pair(320, 180), Pair(360, 640), Pair(180, 640),
+        Pair(480,480), Pair(240, 240))
 
 fun Pair<Int, Int>.variance(other: Pair<Int, Int>): Int = abs(this.first - other.first) + abs(this.second - other.second)
 
 fun Pair<Int, Int>.fitWithin(): Pair<Int, Int>{
-    val maxWidth = 640
-    val maxHeight = 360
+    val maxDimension = 640
```

Comment:
Just to make this function more useful and a bit clearer, how about making maxDimension as a parameter with a default value. E.g. 

```
fun Pair<Int, Int>.fitWithin(maxDimension: Int = 640)
```

Reply:
done